build/common.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash -e
```

Comment:
Are these files coming from the submodule, or are they being checked in directly?

Reply:
They were checked in directly but should be updated now :+1: CI running :heavy_check_mark:

Reply:
@negz could you add the golangci bot to this repo as well?

Reply:
Done, I believe.